docs/Test-Runner.md

```diff
@@ -0,0 +1,9 @@
+# Elección Test Runner  
+
+Como se ha comentado en el issue [#19](https://github.com/alvarogaro/AGR-MVS/issues/19), vamos a realizar una elección del Test Runner que vamos a utilizar en nuestro proyecto, para ello vamos a tener como criterios, las mejores prácticas para el lenguaje, la puntuación de la herramienta en la web [Snyk](https://snyk.io/), el mantenimiento y comunidad de la herramienta y por último su popularidad.
+
+* Pytest: Es uno de los Test-Runner mas populares de Python con una gran comunidad de mantenimiento, tiene un 97 de puntuación en Snyk, no hace uso de entornos virtuales para la realización de los test.
+
+* Nose2: Tiene una gran comunidad de mantenimiento y cierta popularidad. En Snyk, obtiene una puntuación de 85, no hace uso de entornos virtuales para la realización de los test.
```

Comment:
No pongáis información irrelevante. ¿De qué sirve que use entornos virtuales?

Reply:
Cierto, lo he eliminado como criterio de elección